DigitalLearningSolutions.Web/Styles/shared/searchableElements/itemsPerPage.scss

```diff
@@ -47,7 +47,13 @@
 .items-per-page-submit {
   @extend .right-submit-button;
   background-color: $button-dark-blue;
-  
+
+  #area-to-hide-while-loading & {
+    .js-enabled & {
+      display: none;
+    }
+  }

```

Comment:
We can't use the no-js-only class because we need the styling while js is enabled, but only when we have javascript filters etc, so yeah.
I've tried to come up with a name for both in js-styling-hidden-area-while-loading